src/main/webapp/index.html

```diff
@@ -53,10 +53,14 @@ <h1>How does it work?</h1>
   </p>
   <p>
     <strong class="list-mark">3.</strong>
-    Download image with blurred faces.
+    If you want to make some blurred area visible (e.g. to blur all your friends in the photo, but not yourself), simply click on that area in preview image.
   </p>
   <p>
     <strong class="list-mark">4.</strong>
+    Download image with blurred faces.
```

Comment:
It's no longer only 'faces', right?

Reply:
Here we didn't change the UI for options yet, so it is only faces. I guess this change is done somewhere in other PRs.

Reply:
I see, got lost in versions and PRs :)